app/settings/language.tsx

```diff
@@ -0,0 +1,75 @@
+import { router } from 'expo-router'
+import React, { Fragment } from 'react'
+import { useTranslation } from 'react-i18next'
+import { View } from 'react-native'
+import { useMMKVString } from 'react-native-mmkv'
+
+import Divider from '@/components/shared/Divider'
+import FlexRow from '@/components/shared/FlexRow'
+import Icon from '@/components/shared/Icon'
+import PressableStyled from '@/components/shared/PressableStyled'
+import ScreenContent from '@/components/shared/ScreenContent'
+import ScreenView from '@/components/shared/ScreenView'
+import Typography from '@/components/shared/Typography'
+import { useTranslation as useTranslationLocal } from '@/hooks/useTranslation'
+import { STORAGE_LANGUAGE_KEY } from '@/utils/langueDetectorPlugin'
+import { storage } from '@/utils/mmkv'
+
+// TODO
+const Page = () => {
+  const t = useTranslationLocal('Settings')
+  const { i18n } = useTranslation()
+  const [mmkvLocale] = useMMKVString(STORAGE_LANGUAGE_KEY)
+
+  const setLanguage = async (language: 'sk' | 'en') => {
+    await i18n.changeLanguage(language)
+
```

Comment:
Doesn't the `mmkvLocale` have to be updated? The language changes, but the UI shows always only "System"

Reply:
Yes, thanks!